src/include/target/BETAFPV_2400_TX.h

```diff
@@ -1,5 +1,5 @@
 #ifndef DEVICE_NAME
-#define DEVICE_NAME          "BETAFPV 2400TX"
+#define DEVICE_NAME          "bFPV 2400TX"
```

Comment:
If you do choose to use freq domain i'd prefer it being consistent throughought: `bFPV 2G4TX`